providers/aws/components/baseline/setup.ftl

```diff
@@ -472,9 +472,15 @@
                                         "    ;;",
                                         "  create|update)",
                                         "    info \"Removing legacy oai credential\"",
-                                        "    delete_oai_credentials" + " " +
+                                        "    if is_oai_credential_used" + " " +
                                                "\"" + regionId + "\" " +
-                                               "\"" + legacyOAIName + "\" || return $?",
+                                               "\"" + legacyOAIName + "\"; then",
+                                        "      warn \"Legacy OAI in use - it will need to be manually deleted ...\"",
```

Comment:
"it will be deleted once it is no longer in use"

Reply:
Is there some cleanup logic somewhere? The AOI cleanup code from baseline is removed once the OAI pseudo stack is deleted.